csharp/src/Game.Api/Controllers/GamesController.cs

```diff
@@ -0,0 +1,85 @@
+using AutoMapper;
+using Game.Services.Interfaces;
+using api.ViewModels;
+using Microsoft.AspNetCore.Mvc;
+
+namespace api.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public partial class GamesController : ControllerBase
+    {
+        private readonly IGameService _gameService;
+        private readonly IMapper _mapper;
+
+        public GamesController(IGameService gameService, IMapper mapper)
+        {
+            _gameService = gameService;
+            _mapper = mapper;
+        }
+
+        [HttpPost]
+        public ActionResult<Guid> CreateGame()
+        {
+            var newGameId = _gameService.CreateGame();
+            var gameDto = _gameService.GetGame(newGameId);
+            var response = _mapper.Map<CreateGameViewModel>(gameDto);
+
+            return CreatedAtAction(nameof(GetGame), new { gameId = newGameId }, response);
```

Comment:
This won't match the payload expected from the Requirements; so the API tests won't pass.